test/e2e/specs/block-mover.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import '../support/bootstrap';
+import { newPost, newDesktopBrowserPage } from '../support/utils';
+
+describe( 'block mover', () => {
+	beforeEach( async () => {
+		await newDesktopBrowserPage();
```

Comment:
Normally we open a page for all the tests e.g: await newDesktopBrowserPage(); is in a before all block and not in a beforeEach block. Was there any special reason for this difference? Can the newDesktopBrowserPage call be in a beforeAll block?

Reply:
Probably not but I'll double-check.

Reply:
Ah, scratch that; we need an empty post for each test. If we use `beforeAll` we don't have a clean slate that we need to know how many blocks are there.

Our tests in the past use `beforeAll` possibly for speed? The globals in the tests are kinda weird... anyway, this needs a fresh state for each test so `beforeEach` it is...

Reply:
But if we use newDesktopBrowserPage in beforeAll and newPost in beforeEach don't we get a fresh state on each test? We will just reuse the same tab but we start with an empty post on each test.

Reply:
I tried that out locally and it seemed to hang. I'll give it another go though, I just realised my environment could have been messed up locally from an unstashed, unrelated change.